rotkehlchen/tests/api/test_assets.py

```diff
@@ -262,3 +262,133 @@ def test_ignored_assets_endpoint_errors(rotkehlchen_api_server_with_exchanges, m
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert len(result['entries']) == 20
+    assert 'entries' in result
+    assert 'entries_found' in result
+    assert 'entries_total' in result
+    assert 'entries_limit' in result
+
+    # check that providing multiple order_by_attributes fails
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'allassetsresource',
+        ),
+        json={
+            'limit': 20,
+            'offset': 0,
+            'asset_type': 'fiat',
+            'order_by_attributes': ['name', 'symbol'],
+            'ascending': [True, False],
+        },
+    )
+    assert_error_response(response, contained_in_msg='Multiple fields ordering is not allowed.')
+
+
+def test_get_assets_mappings(rotkehlchen_api_server):
+    """Test that providing a list of asset identifiers, the appropriate assets mappings are returned."""  # noqa: E501
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsmappingresource',
+        ),
+        json={'identifiers': ['BTC', 'TRY', 'EUR']},
+    )
+    result = assert_proper_response_with_result(response)
+    assert len(result) == 3
+    for entry in result:
+        assert entry['identifier'] in ('BTC', 'TRY', 'EUR')
+
+    # check that providing multiple order_by_attributes fails
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsmappingresource',
+        ),
+        json={'identifiers': ['BTC', 'TRY', 'invalid']},
+    )
+    assert_error_response(response, contained_in_msg='One or more identifiers do not exist')
+
+
+def test_search_assets(rotkehlchen_api_server):
+    """Test that searching for assets using a keyword works."""
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetssearchresource',
+        ),
+        json={
+            'substring_search': 'Bitcoin',
+            'limit': 50,
+            'order_by_attributes': ['name'],
+            'ascending': [True],
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    for entry in result:
+        assert 'bitcoin' in entry['name'].lower() or entry['symbol'].lower()
+
+    # check that searching for a non-existent asset returns nothing
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetssearchresource',
+        ),
+        json={
+            'substring_search': 'idontexist',
+            'limit': 50,
+            'order_by_attributes': ['name'],
+            'ascending': [True],
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert len(result) == 0
+
+
+def test_search_assets_with_levenshtein(rotkehlchen_api_server):
+    """Test that searching for assets using a keyword works(levenshtein approach)."""
+    response = requests.post(
```

Comment:
You haven't tried for searches that return multiple possible results right? for example searching for `eth`

Reply:
I did. Just didn't add it to the test query. Will do that now